package.json

```diff
@@ -4,13 +4,14 @@
 	"description": "",
 	"main": "index.js",
 	"scripts": {
-		"start": "webpack-dev-server --config webpack/webpack.dev.js",
-		"test": "jest",
-		"build": "webpack --config webpack/webpack.prod.js",
-		"build:dev": "webpack --config webpack/webpack.dev.js",
-		"tslint-prettier": "tslint-config-prettier-check ./tslint.json",
+		"start": "webpack-dev-server --config config/webpack/webpack.dev.js",
+		"build": "webpack --config config/webpack/webpack.prod.js",
+		"build:dev": "webpack --config config/webpack/webpack.dev.js",
+		"test": "cross-env NODE_ENV=test jest",
```

Comment:
Jest will set process.env.NODE_ENV to 'test' if it's not set to something else. You can use that in your configuration to conditionally setup only the compilation needed for Jest, e.g.

https://jestjs.io/docs/en/getting-started#using-babel

Reply:
Might be redundant. But still LGTM